Make Credo.Check.Warning.ForbiddenModule work for grouped aliases

hoyon opened the pull request:
Fixes #946 

Makes the `Credo.Check.Warning.ForbiddenModule` check work when the forbidden module is aliased in a grouped alias like:

```elixir
defmodule CredoSampleModule do
    alias CredoSampleModule.{ForbiddenModule}
    def some_function, do: ForbiddenModule.another_function()
end
```

rrrene commented on test/credo/check/warning/forbidden_module_test.exs:
```diff
@@ -65,6 +65,18 @@ defmodule Credo.Check.Warning.ForbiddenModuleTest do
+  test "it should report on grouped aliases" do
+    """
+    defmodule CredoSampleModule do
+      alias CredoSampleModule.{ForbiddenModule}
```
```suggestion
      alias CredoSampleModule.{AllowedMod, ForbiddenModule, ForbiddenModule2}
```

rrrene commented on test/credo/check/warning/forbidden_module_test.exs:
```diff
@@ -65,6 +65,18 @@ defmodule Credo.Check.Warning.ForbiddenModuleTest do
+    end
+    """
+    |> to_source_file
+    |> run_check(@described_check, modules: [CredoSampleModule.ForbiddenModule])
```
```suggestion
    |> run_check(@described_check, modules: [CredoSampleModule.ForbiddenModule, CredoSampleModule.ForbiddenModule2])
```

rrrene commented on test/credo/check/warning/forbidden_module_test.exs:
```diff
@@ -65,6 +65,18 @@ defmodule Credo.Check.Warning.ForbiddenModuleTest do
+    """
+    |> to_source_file
+    |> run_check(@described_check, modules: [CredoSampleModule.ForbiddenModule])
+    |> assert_issue()
```
```suggestion
    |> assert_issues()
```

rrrene reviewed (commented):
Great. I attempted a fix for this, but I think yours is more suitable.

One suggestion: We should check that the check is able to find multiple forbidden modules in a multi alias call.

hoyon commented:
> Great. I attempted a fix for this, but I think yours is more suitable.
> 
> One suggestion: We should check that the check is able to find multiple forbidden modules in a multi alias call.

I've implemented your suggestion and fixed another bug when there's multiple forbidden modules.

rrrene commented:
@hoyon Thx!
